torchsummary/torchsummary.py

```diff
@@ -7,11 +7,20 @@
 
 
 def summary(model, input_size, batch_size=-1, device=torch.device('cuda:0'), dtypes=None):
+    result, params_info = summary_string(
+        model, input_size, batch_size, device, dtypes)
+    print(result)
+
+    return params_info
```

Comment:
In my changes, the original `summary` function doesn't change return variables. Instead, I added `summary_string` which can be selectively used by user that requires string result. 

So the question is, should I add a new test for `summary_string`, or change the functionality of existing `summary` function to return result string instead of printing? I'll follow the owner's opinion :)

Reply:
Ahh, I see now what you mean! Yes, there is no need then really for a separate test for summary_string I think!